.prettierrc

```diff
@@ -1,5 +1,13 @@
 {
     "singleQuote": true,
     "tabWidth": 4,
-    "printWidth": 120
+    "printWidth": 120,
+    "overrides": [
+        {
+            "files": "*.md",
+            "options": {
+                "tabWidth": 2
```

Comment:
I had deleted this one in https://github.com/mdn/interactive-examples/pull/2288/files#diff-663ade211b3a1552162de21c4031fcd16be99407aae5ceecbb491a2efc43d5d2L3 becasue Prettier will defer to EditorConfig when it is already set

Reply:
It looks like we have a mix of 4 and 2 tabwidth across files in this repo, should this be uniform?

Reply:
Probably, I've got some open PRs for ESLint/Stylelint to defer to those tools, that should then defer back to Prettier. I think when it's setup a cleanup could be made back to 2 spaces that is used in the main mdn/content

Reply:
Sounds good, tnx 👍🏻